Source/View Controllers/TCCProfileViewController.swift

```diff
@@ -351,14 +351,26 @@ class TCCProfileViewController: NSViewController {
     func insetIntoAppleEvents(_ executable: Executable) {
```

Comment:
Just noticed this should be renamed to `insertIntoAppleEvents` (missing the `r` currently).

Here's a chance to try out Xcode's Refactor > Rename functionality.  Select this function name, right click, and choose Refactor > Rename... to make sure it's updated everywhere it is used.